projects/packages/my-jetpack/_inc/components/connected-product-card/index.jsx

```diff
@@ -32,7 +32,7 @@ const ConnectedProductCard = ( { admin, slug, showDeactivate } ) => {
 		<ProductCard
 			name={ name }
 			description={ description }
-			status={ status }
+			status={ props.status || status }
```

Comment:
I don't get why this is necessary; no one who uses `ConnectedProductCard` is passing a status.

Reply:
I'm using this change to improve the story experience.

Reply:
Maybe this is not a case to highlight this? And avoid someone forcing some status down.

Reply:
I agree with Renato here. Also if you're using only `status`, why do the whole `...props` ?

Reply:
oh right